src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import NavBar from './Pages/NavBar';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Please add mission, rocket, and myProfile components to your project, These components will be responsible for showing details of rockets and missions.